src/stacks-editor/editor.ts

```diff
@@ -223,6 +223,16 @@ export class StacksEditor implements View {
         const menuContainerFn = () => menuTarget;
         this.options.menuParentContainer = menuContainerFn;
 
+        // create specific area for the editor menu
+        const previewTarget = document.createElement("div");
+        previewTarget.className = "py16";
+        this.target.appendChild(previewTarget);
+
+        // set the editors' preview containers to be the combo container
```

Comment:
@b-kelly @dancormier I think this code should be run only optionally, when the preview feature is enabled.

Currently there is a small issue with this div being added and taking some space, even if it's empty:
https://stackoverflow.atlassian.net/browse/PUBPLAT-3427

Alternatively we could adjust the styles of this div. What do you think?

Reply:
Thanks for the callout. We're currently tweaking how the preview is shown as part of the work for #150, so I'll ensure this is part of the acceptance criteria for that work.